log.go

```diff
@@ -31,6 +34,22 @@ func (r *ResponseRecorder) WriteHeader(n int) {
 	r.response = n
 }
 
+func (r *ResponseRecorder) Hijack() (net.Conn, *bufio.ReadWriter, error) {
+	hijacker, ok := r.ResponseWriter.(http.Hijacker)
+	if !ok {
+		return nil, nil, fmt.Errorf("Not a Hijacker: %T", r.ResponseWriter)
+	}
+	return hijacker.Hijack()
+}
+
+func (r *ResponseRecorder) Flush() {
+	flusher, ok := r.ResponseWriter.(http.Flusher)
+	if !ok {
```

Comment:
Yep, I could also do that. I mainly wrote it this way because I had just writtten Hijack which has the same form..